openreview/venue/venue.py

```diff
@@ -138,6 +139,13 @@ def get_recommendation_id(self, committee_id=None):
             committee_id = self.get_reviewers_id()
         return self.get_invitation_id('Recommendation', prefix=committee_id)
 
+    def get_rebuttal_invitation_id(self, name, number, signature = None):
```

Comment:
why don't you use `get_invitation_id` with the signature as a prefix?

Reply:
`get_invitation_id()` expects either prefix or number I believe, because it overwrites the invitation_id everytime:

```
if prefix:
    invitation_id = prefix
if number:
    invitation_id = f'{self.get_paper_group_prefix(number)}/-/'
```

Reply:
isn't this the same you are doing?

Reply:
similar, but I am giving priority to the prefix, because when it is a signature, it includes the paper number as well. 

`get_rebuttal_invitation_id()` => '${2/content/noteSignatures/value}/-/Rebuttal'
`get_invitation_id()` => 'ICML.cc/2023/Conference/Submission${2/content/noteNumber/value}/-/Rebuttal' (it ignores the prefix because there is a paper number)

Reply:
nvm I see it now, I need to pass prefix only and not number! will update